app/assets/javascripts/components/ops/tenant-component.js

```diff
@@ -0,0 +1,76 @@
+ManageIQ.angular.app.component('tenantComponent', {
+  bindings: {
+    id: '=?',
+    divisible: '=',
+    parent: '=?',
+    angularForm: '=',
```

Comment:
Done in https://github.com/ManageIQ/manageiq-ui-classic/pull/1544/commits/e7a4f4d36fe30f2b9141548389afe7bc586c05d7